sdk/appconfiguration/azure-spring-cloud-feature-management/README.md

```diff
@@ -267,4 +267,4 @@ Options are available to customize how targeting evaluation is performed across
 ## Contributing
 
 <!-- Links -->
-[example_project]: https://github.com/Azure-Samples/azure-spring-boot-samples/tree/azure-spring-boot_3.7/appconfiguration/feature-management-web-sample
+[example_project]: https://github.com/Azure-Samples/azure-spring-boot-samples/tree/tag_azure-spring-boot_3.x/appconfiguration/feature-management-web-sample
```

Comment:
`3.x` has a big scope. How about renaming `tag_azure-spring-boot_3.x` to `tag_azure-spring-boot_3.6.0`?

Reply:
updated @chenrujun